steel-research-international.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Steel Research International</title>
+    <title-short>Steel Res. Int.</title-short>
+    <id>http://www.zotero.org/styles/steel-research-international</id>
+    <link href="http://www.zotero.org/styles/steel-research-international" rel="self"/>
+    <link href="https://onlinelibrary.wiley.com/page/journal/1869344x/homepage/author-guidelines" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-national-library-of-medicine" rel="template"/>
+    <author>
+      <name>Christian Schubert</name>
+    </author>
+    <contributor>
+      <name>Christian Schubert</name>
+      <email>5238962+c-schubert@users.noreply.github.com</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <eissn>1869-344X</eissn>
+    <issnl>1611-3683</issnl>
+    <summary>Citation style for the Steel research international
+    
+    There are some difficulties i.e. with patents, preprint  specifications and the variables that the CSL offers (nor preprint type or company variable availably), which probably cannot be fixed right now (perfectly) and need manual correction.
+    Also there is no way to automatically use or define journal or conference abbrevitation in the csl file right now?</summary>
+    <updated>2021-11-10T16:22:30+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-US">
+    <date form="text" delimiter=" ">
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year" font-weight="bold"/>
+    </date>
+    <terms>
+      <term name="chapter" form="short">Ch.</term>
+      <term name="editor" form="short">eds</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
```

Comment:
You'll want to add a label here (which would appear for editors)

Reply:
Yes I probably do. There is no real default for this case. So for now I use what seems to make sense to me and that would be to use `Eds:` as a prefix, as it is also given when author and editor are specified.

Reply:
No, don't prefix since this starts the bibliography entry and would look like it's out of alphabetical order. Just using label after `name`, either in parentheses or preceded by a comma, works

Reply:
There should be no alphabetical order in the bibliography, instead it should be in citation order, so this should be no problem. However, I will change it as you proposed (I hope I got it right..), because it looks little more consistent.